README.md

```diff
@@ -5,39 +5,363 @@
+
+#### Aggregating Community Issues
+
+To see open community issues on a repository aggregated by a Trello board, click on a link under the "Issues" column in the subscriptions table:
+
+![Subscriptions Issues Link](https://github.com/DataDog/gello/blob/master/images/subscriptions_highlight_issues.png)
+
+Here there's a paginated listing of all community issues still open, with links to the corresponding GitHub url, and Trello url (for the card created by _Gello_):
+
+![Community Issues By Board](https://github.com/DataDog/gello/blob/master/images/community_issues_by_board.png)
+
+#### Aggregating Community Pull Requests
+
+Likewise, to see open community pull requests on a repository aggregated by a Trello board, click on a link under the "Pull Requests" column in the subscriptions table:
+
+![Subscriptions Issues Link](https://github.com/DataDog/gello/blob/master/images/subscriptions_highlight_pull_requests.png)
+
+
+## How It Works
+
+_Gello_ works by _subscribing_ Trello boards to GitHub repositories through a web UI, and selecting the corresponding Trello lists you would like cards to be created to following a GitHub event.
+
+### GitHub Webhooks
+_Gello_ uses [GitHub webhooks](https://developer.github.com/webhooks/) to get event updates from the repositories subscribed.
+
+When a subscription is made, _Gello_ will create a GitHub webhook on the corresponding repository with the following permissions: `Issues`, `Issue comments`, `Pull requests`, and `Pull request review comments`.
+
+This will allow the server to receive webhook events whenever an issue or pull request is created or commented upon. They are necessary for both the _autocard_ and _manual_ settings.
+
+### GitHub Events
+#### Pull Request Event
+
+For the _autocard_ setting, if a pull request is created by a person outside of the members of a GitHub organization on a subscribing repository, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Issue Event
+
+Likewise, for the _autocard_ setting, if an issue is created by a person outside of the members of a GitHub organization on a subscribed repository, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Pull Request Comment Event
+
+For the _manual_ setting, if a pull request has been created, and a member of the GitHub organization comments `gello create_card` on the pull request, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+#### Issue Comment Event
+
+For the _manual_ setting, if an issue has been created, and a member of the GitHub organization comments `gello create_card` on the issue, a Trello card will be submitted to a board list (or a number of lists), configurable in the _Gello_ web-UI.
+
+
+## Configuration
```

Comment:
I've thought about this, and the way I envision this doc is in 3 main sections:
	- what Gello does (the why & detailed features)
	- Development (with local/ containerized db & redis)
	- Deployment (with Heroku)
"Configuration" here is mainly environment variables - we could put it in a separate .md and link it in both Development and Deployment? Doing this would make the docs shorter & more logical too.

Reply:
This approach is 👍